snapshots/gapminder/2023-09-18/under_five_mortality.xlsx.dvc

```diff
@@ -0,0 +1,30 @@
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Child Mortality Rate (Gapminder)
```

Comment:
The title should not mention the producer. Shouldn't the data product be `Child mortality rate under age five`?